ProjetSupaero2018/catkin_ws/src/display/models/basic_car/model.sdf

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" ?>
+<sdf version="1.5">
+	<model name="basic_car">
+	  <link name="sphere">
+		<pose>0 0 1 0 0 0</pose>
+		
+		<inertial>
+			<mass>1</mass>
+			<inertia>
+				<ixx>0.4</ixx> <!-- inertia is 2/5*M*R2 -->
+				<iyy>0.4</iyy>
+				<izz>0.4</izz>
+				<ixy>0</ixy>
+				<ixz>0</ixz>
+				<iyz>0</iyz>
+			</inertia>
+		</inertial>
+				
+		<collision name="car_collision">
+		  <geometry>
+			<sphere>
+				<radius>1</radius>
+			</sphere>
+		  </geometry>
+		  
+		  <!--
+		  <surface>
+		  	<friction>
```

Comment:
I don't think we need a friction model either

Reply:
This one is not

Reply:
Did you comment it, or did I ?

Reply:
I didn't commit anything on this branch, so I assume you did. If it's inside a comment block, it's really useless then :P

Reply:
Yay !

Reply:
Mom, can I keep it ? So that we don't have to look for it later on, if needed